src/main.asm

```diff
@@ -63,14 +43,24 @@ html:
     db "    <h2>Rush B</h2>", 10
     db "  </body>", 10
     db "</html>", 10, 0
-html_size:
-    dq 0
+html_size: equ $-html-1
+css:
+    db "body { background: black }", 10, 0
+css_size:   equ $-css-1
 http:
     db "HTTP/1.1 200 OK", 13, 10
     db "Content-Type: text/html", 13, 10
```

Comment:
We use the same HTTP template for CSS content. But CSS has `text/css` Content-Type. Make content type of http template configurable.